status/projects.py

```diff
@@ -1049,3 +1049,104 @@ def post(self, project_id):
             self.set_status(200)
             self.set_header("Content-type", "application/json")
             self.write(self.request.body)
+
+
+class PrioProjectsTableHandler(SafeHandler):
+    """Serves information about projects statuses.
+
+    Loaded through /api/v1/prio_projects"""
+
+    def get(self):
+        from collections.abc import Iterable
+        def flatten(l):
+            for el in l:
+                if isinstance(el, Iterable) and not isinstance(el, str):
+                    yield from flatten(el)
+                else:
+                    yield el
+
+        projects = {}
+        def_dates_rec_ctrl = { 'days_recep_ctrl' : ['open_date', 'queued']
+                             }
+        #dates in order
+        def_dates_ongoing = { 'days_prep_start' : ['queued', 'library_prep_start'],
+                               'days_prep' : ['library_prep_start', 'qc_library_finished'],
+                               'days_seq_start' : [['qc_library_finished', 'queued'], 'sequencing_start_date'],
+                               'days_seq' : ['sequencing_start_date', 'all_samples_sequenced'],
+                               'days_analysis' : ['all_samples_sequenced', 'best_practice_analysis_completed'],
+                               'days_data_delivery' : ['best_practice_analysis_completed', 'all_raw_data_delivered'],
+                               'days_close' : ['all_raw_data_delivered', 'close_date']
+                             }
+
+        date_fields = list(flatten(def_dates_ongoing.values()))
```

Comment:
If this is the only place you use flatten, maybe you can just write `def_dates_ongoing` the way you want it right away?